### PR TITLE
fix handleFee logic in KyberNetwork and added iFeeHandler interface

### PR DESCRIPTION
**Pull request by STYJ**
Had to change some logic in my handleFees so the handleFee function call doesn't throw a revert in Desmond's test

### CODE REVIEW

File: contractsV5/KyberNetwork.sol [hunk 1]
```diff
@@ -730,18 +722,30 @@ contract KyberNetwork is Withdrawable, Utils, IKyberNetwork, ReentrancyGuard {
...
+        // fees should add up to 100%.
+        address[] memory eligibleWallets = new address[](tradeData.numFeePayingReserves);
+        uint[] memory rebatePercentages = new uint[](tradeData.numFeePayingReserves);
+
+        // Updates reserve eligibility and rebate percentages
+        updateEligibilityAndRebates(eligibleWallets, rebatePercentages, tradeData);
+
+        // // Send total fee amount to fee handler with reserve data.
+        // require(
+        //     feeHandlerContract.handleFees.value(tradeData.networkFeeWei)(eligibleWallets, rebatePercentages),
```
Comment (ilanDoron) [inline contractsV5/KyberNetwork.sol]: this is commented out.
need to send. no?

File: contractsV5/IFeeHandler.sol [hunk 2]
```diff
@@ -1,6 +1,5 @@
 pragma solidity 0.5.11;
 
-
 interface IFeeHandler {
-    function handleFee() external returns(bool);
-}
+    function handleFees(address[] calldata eligibleWallets, uint[] calldata rebatePercentages) external payable returns(bool);
+}
```
Comment (ilanDoron) [inline contractsV5/IFeeHandler.sol]: please CR LF

File: contractsV5/KyberNetwork.sol [hunk 3]
```diff
@@ -730,18 +722,30 @@ contract KyberNetwork is Withdrawable, Utils, IKyberNetwork, ReentrancyGuard {
...
 
-        // Send total fee amount to fee handler with reserve data.
-        require(
-            feeHandlerContract.handleFees.value(tradeData.networkFeeWei)(eligibleWallets, rebatePercentages),
-            "Transfer network fee to FeeHandler failed"
-        );
+        //no need to handle fees if no fee paying reserves
+        if (tradeData.numFeePayingReserves == 0) return true;
+
+        // create array of reserves receiving fees + fee percent per reserve
```
Comment (ilanDoron) [inline contractsV5/KyberNetwork.sol]: fix comment.
create array of rebate wallets

File: contractsV5/KyberNetwork.sol (hunk 1 above)
Comment (STYJ) [reply contractsV5/KyberNetwork.sol]: Yes, sorry I just took his code and pasted it. Will uncomment n fix

File: contractsV5/IFeeHandler.sol (hunk 2 above)
Comment (STYJ) [reply contractsV5/IFeeHandler.sol]: noted

File: contractsV5/KyberNetwork.sol (hunk 3 above)
Comment (STYJ) [reply contractsV5/KyberNetwork.sol]: done

**Pull request merged**